conanfile.py

```diff
@@ -45,19 +46,7 @@ class Libnest2DConan(ConanFile):
     }
 
     def layout(self):
-        self.folders.source = "."
-        try:
-            build_type = str(self.settings.build_type)
-        except ConanException:
-            raise ConanException("'build_type' setting not defined, it is necessary for cmake_layout()")
-        self.folders.build = f"cmake-build-{build_type.lower()}"
-        self.folders.generators = os.path.join(self.folders.build, "conan")
-
-        self.cpp.source.includedirs = ["include"]
-
-        self.cpp.build.libdirs = ["."]
-        self.cpp.build.bindirs = ["."]
-
+        cmake_layout(self)
```

Comment:
Don't we need to add:
```
self.cpp.package.libs = [f"libnest2d_{self.options.optimizer}_{self.options.the_other_thing}"]
```

Since libnest2d can be build either as: `shared`, `static` or as `header_only`

Reply:
If we define it here, we also have to revisit this piece of code:
```
    def package_info(self):
        if not self.options.header_only:
            self.cpp_info.libs = tools.collect_libs(self)
```

Reply:
I'm not really sure how to implement this suggestion. I couldn't really find an example (either online or within one of our own repo's) how where something like`[f"libnest2d_{self.options.optimizer}_{self.options.the_other_thing}"]` is used. I imagine this is to differentiate between binaries that are build using the different build options?

side note, with
```py
self.cpp.package.libs = [f"libnest2d_{self.options.optimizer}_{self.options.the_other_thing}"]
```
you mean
```py
self.cpp.package.libs = [f"libnest2d_{self.options.geometries}_{self.options.optimizer}"]
```
right?